.travis.yml

```diff
@@ -1,5 +1,10 @@
 if: (branch = master) OR (type = pull_request) OR (tag IS present)
+stages:
+  - name: "Build and Test"
+  - name: "Deploy"
+    if: tag IS present
 
+stage: "Build and Test"
```

Comment:
Do we need this stage name twice? It's also in line 24.

Reply:
Don't see why we would, giving it a try.